samples/ehr-flask/katanemo/integrations/flask_auth.py

```diff
@@ -0,0 +1,69 @@
+import json
+import katanemo_sdk
+import katanemo_auth_sdk
+import requests
+from flask import request as flask_request
+
+class KatanemoFlaskAuth:
+    
+    def __init__(self, app):
+        self.app = app
+        self.api_client = None
+        self.auth_client = None
+        self.api_access_token = None
+        self.client_id = None
+
+    def register(self, client_name, service_id, client_id, client_secret):
+        self.client_id = client_id
+        self.client_secret = client_secret
+        access_api_client = katanemo_sdk.AccessControlApi(self._get_access_api_client())
+        req = katanemo_sdk.OAuthTokenRequest(clientId=client_id, clientSecret=client_secret)
+        resp = access_api_client.get_short_term_token(req)
```

Comment:
Its unclear why we use access_api_client in some instances and katanemo_sdk in some other instances. Why isn't it that we build our client once (initialized with all the relevant values) and then use that to make the necessary calls like OAuthTokenRequest or GetShortTermToken, etc

Reply:
access_api_client is local variable for AccessControlApi from katanemo_sdk. That is the pattern employed by openapitools python client.